### PR TITLE
feat(messaging): add api-cli docs for sns/sqs aws cli and python/node sdks

### PR DESCRIPTION
**Pull request by RoRoJ**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Add Messaging API/CLI doc for SNS/SQS

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: compute/messaging/api-cli/connect-aws-cli.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+2. Open the `~/.aws/config` file you just created in a text editor and edit it as follows:
+
+  ```
+  [plugins]
+  endpoint = awscli_plugin_endpoint
+
+  [default]
+  sns =
+	endpoint_url= http://51.159.73.139:4280
```
Comment (JulienVdG) [inline compute/messaging/api-cli/connect-aws-cli.mdx:60]:
```suggestion
	endpoint_url = http://sqs-sns.mnq.fr-par.scw.cloud
```

We just added dns entries, better use them ;-)

File: compute/messaging/api-cli/connect-aws-cli.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  sqs =
+	endpoint_url= http://51.159.73.139:4280
```
Comment (JulienVdG) [inline compute/messaging/api-cli/connect-aws-cli.mdx:62]:
```suggestion
	endpoint_url = http://sqs-sns.mnq.fr-par.scw.cloud 
```
same.

File: compute/messaging/api-cli/connect-aws-cli.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  
+  ``` 
+  [plugins]
+  endpoint = awscli_plugin_endpoint
+
+  [default]
+  sns =
+	endpoint_url= http://51.159.73.139:4280
+  sqs =
+	endpoint_url= http://51.159.73.139:4280
```
Comment (JulienVdG) [inline compute/messaging/api-cli/connect-aws-cli.mdx:75]:
```suggestion
	endpoint_url = http://sqs-sns.mnq.fr-par.scw.cloud
  sqs =
	endpoint_url = http://sqs-sns.mnq.fr-par.scw.cloud
```

File: compute/messaging/api-cli/python-node-sqs.mdx [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+sqs = boto3.resource('sqs',
+  endpoint_url=[],
+  aws_access_key_id=[],
+  aws_secret_access_key=[],
+  region_name='fr-par')
+```
+
+<Message type="note">
+
+  The `endpoint_url` for Scaleway Messaging is `http://51.159.73.139:4280`. For the access and secret key values, use the credentials you [generated](/compute/messaging/how-to/create-credentials/) for your SNS/SQS namespace.
```
Comment (JulienVdG) [inline compute/messaging/api-cli/python-node-sqs.mdx:45]:
```suggestion
  The `endpoint_url` for Scaleway Messaging is `http://sqs-sns.mnq.fr-par.scw.cloud`. For the access and secret key values, use the credentials you [generated](/compute/messaging/how-to/create-credentials/) for your SNS/SQS namespace.
```

File: compute/messaging/api-cli/python-node-sqs.mdx [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+  endpoint: [],
+  accessKeyId: [],
+  secretAccessKey: [],
+  region: 'fr-par'
+  });
+```
+
+<Message type="note">
+
+  The `endpoint_url` for Scaleway Messaging is `http://51.159.73.139:4280`. For the access and secret key values, use the credentials you [generated](/compute/messaging/how-to/create-credentials/) for your SNS/SQS namespace.
```
Comment (JulienVdG) [inline compute/messaging/api-cli/python-node-sqs.mdx:95]:
```suggestion
  The `endpoint_url` for Scaleway Messaging is `http://sqs-sns.mnq.fr-par.scw.cloud`. For the access and secret key values, use the credentials you [generated](/compute/messaging/how-to/create-credentials/) for your SNS/SQS namespace.
```

**Review by JulienVdG — COMMENTED** [review]
Nice work !
We now have a dns, I sugested changes to the ip addresses. I hope I didn't miss some.

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by lucasscw — APPROVED** [review]

**Review by lucasscw — APPROVED** [review]

**Pull request merged**